src/WebApp/Pages/Questions.cshtml

```diff
@@ -0,0 +1,63 @@
+﻿@page "/{category}/questions"
+@model Codidact.Core.WebApp.Pages.Posts.QuestionsModel
+@{
+    ViewData["Title"] = "Questions";
+}
+
+<partial name="_ValidationSummary" />
+
+
+<div class="grid has-margin-4">
+    <div class="grid--cell is-flexible">
+        <h1 class="has-margin-0">Q&amp;A posts</h1>
+    </div>
+    <div class="grid--cell">
+        <a href="newpost.html" class="button is-medium has-margin-4 is-filled">New question</a>
+    </div>
+</div>
+<div class="button-list is-gutterless has-margin-top-4 has-float-right">
+    <button class="button is-muted is-outlined is-active">best</button>
```

Comment:
These should probably be links changing the ?-part of the URL and hence the query sort order.

Reply:
I changed them to a href with individual sorts